src/test/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/service/LicenceServiceTest.kt

```diff
@@ -1898,6 +1898,12 @@ class LicenceServiceTest {
       probationTeamDescription = "Cardiff South Team A",
       conditionalReleaseDate = LocalDate.of(2021, 10, 22),
       actualReleaseDate = LocalDate.of(2021, 10, 22),
+      sentenceStartDate = LocalDate.of(2018, 10, 22),
+      sentenceEndDate = LocalDate.of(2021, 10, 22),
+      licenceStartDate = LocalDate.of(2021, 10, 22),
+      licenceExpiryDate = LocalDate.of(2021, 10, 22),
+      topupSupervisionStartDate = LocalDate.of(2021, 10, 22),
+      topupSupervisionExpiryDate = LocalDate.of(2021, 10, 22),
```

Comment:
as a tip - its worth having unique values in test data..
It makes accidental typos pop out more

As an example: if you'd accidentally set `topupSupervisionStartDate` and `topupSupervisionEndDate` to the same value in your converter for instance

Reply:
Good point!
I nabbed these from the `aLicenceEntity` above but I probably should've fixed both